crimsobot/cogs/mystery.py

```diff
@@ -58,6 +60,92 @@ async def tarot(self, ctx: commands.Context, spread: str = 'ppf') -> None:
         fp, descriptions = tarot.reading(spread)
         await ctx.send('\n'.join(descriptions), file=discord.File(fp, 'reading.png'))
 
+    @commands.command(aliases=['tarotcard'], brief='Inspect a tarot card for further study.')
+    async def card(self, ctx: commands.Context) -> None:
+        """Inspect a tarot card chosen from the menu for further study.
+        See also >help tarot for more info about the Major Arcana and the four suits."""
+
+        # the suits
+        suits = ['Major arcana', 'Wands', 'Pentacles', 'Cups', 'Swords']
+        suit_list = []
+        for idx, suit in enumerate(suits):
+            suit_list.append('{}. {}'.format(idx, suit))
+
+        # prompt 1 of 2: choose suit
+        embed = c.crimbed(
+            title='Choose a suit:',
+            descr='\n'.join(suit_list),
+            thumb_name='wizard',
+            footer='Type the number to choose.'
+        )
+        prompt_suit = await ctx.send(embed=embed)
+
+        # define check for suit
+        def suit_check(msg: discord.Message) -> bool:
+            try:
+                valid_choice = 0 <= int(msg.content) <= 4
+                in_channel = msg.channel == ctx.message.channel
+                is_author = msg.author == ctx.message.author
+
+                return valid_choice and in_channel and is_author
+
+            except ValueError:
+                return False
+
+        # wait for user to spcify suit
+        try:
+            msg = await self.bot.wait_for('message', check=suit_check, timeout=20)
+        except asyncio.TimeoutError:
+            await prompt_suit.delete()
+            return
+
+        if msg is not None:
+            suit_choice = int(msg.content)
+            await msg.delete()
+            await prompt_suit.delete()
```

Comment:
Let's change this to the following just to be safe. Same around 141-144
```suggestion
        await prompt_suit.delete()

        if msg is None:
            return

        suit_choice = int(msg.content)
        await msg.delete()
```

Reply:
Alright will do.